javascripts/govuk/analytics/sample-init.js

```diff
@@ -0,0 +1,20 @@
+(function() {
```

Comment:
I think this should live in a folder called `/examples` so it's very clear what this is doing